### PR TITLE
Exemption Details

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#17419

*Description of changes:*
- Implements staff and Qs components in the Home Details view for Exemptions
- Minor UX/Styling adjustments to meet current designs and spacing
- Jest Test updates
- Minor AA updates as a result of caught gaps via unit tests

- Figma reference: https://www.figma.com/file/qrCF38xej31oK9dFd8NNun/MHR-Exemptions?type=design&node-id=749-9323&mode=design&t=xLK0XnC2041rPVue-0

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1565-s448vnv5.web.app

File: ppr-ui/jest.config.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ module.exports = {
       babelConfig: true
     }
   },
-  testTimeout: 50000,
+  testTimeout: 500000,
```
Comment (dimak1) [inline ppr-ui/jest.config.js:9]: 😮

**Review by doug-lovett — APPROVED** [review]
Looks great.

File: ppr-ui/tests/jest.setup.ts [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
...VhIiwiYWNyIjoiMCIsImFsbG93ZWQtb3JpZ2lucyI6WyIqIl0sInJlYWxtX2FjY2VzcyI6eyJyb2xlcyI6WyJwdWJsaWNfdXNlciIsInBwciIsImVkaXQiLCJhY2NvdW50X2hvbGRlciIsIm9mZmxpbmVfYWNjZXNzIiwidW1hX2F1dGhvcml6YXRpb24iXX0sInJlc291cmNlX2FjY2VzcyI6eyJhY2NvdW50Ijp7InJvbGVzIjpbIm1hbmFnZS1hY2NvdW50IiwibWFuYWdlLWFjY291bnQtbGlua3MiLCJ2aWV3LXByb2ZpbGUiXX19LCJzY29wZSI6Im9wZW5pZCIsImZpcnN0bmFtZSI6IkJDUkVHMiBTdmV0bGFuYSIsInJvbGVzIjpbInB1YmxpY191c2VyIiwicHByIiwiZWRpdCIsImFjY291bnRfaG9sZGVyIiwib2ZmbGluZV9hY2Nlc3MiLCJ1bWFfYXV0aG9yaXphdGlvbiJdLCJuYW1lIjoiQkNSRUcyIFN2ZXRsYW5hIEZPVVJURUVOIiwiaWRwX3VzZXJpZCI6IkhERjNEWVFFUUhQVU1VT01QUUMyQkFGSVJETFZPV0s2IiwicHJlZmVycmVkX3VzZXJuYW1lIjoiYmNzYy9oZGYzZHlxZXFocHVtdW9tcHFjMmJhZmlyZGx2b3drNiIsImxvZ2luU291cmNlIjoiQkNTQyIsImxhc3RuYW1lIjoiRk9VUlRFRU4iLCJ1c2VybmFtZSI6ImJjc2MvaGRmM2R5cWVxaHB1bXVvbXBxYzJiYWZpcmRsdm93azYifQ.UUamIDN1LWms2oK5YG9yEmfen6ISFoY9AGw7ZJrsmDiElt0XwI_lj6DPYdMieXgXQ4Ji7jRVSMNhX4LfxpC1JipepUbI3kBLf0lelTudhZyD9MOg-VYaLAAEwAY57Z8h7EOCQp0PLS8NAMwNs90t4sJ449uZ3HprEMfMvkaZ0X3Cv495U0m5Qr-GDT7PHeLqkh3297gvxx3PdIGZIWcIwz-lFo8jNYxpEtY1LivZXnCsfrLDEW-vVK5kmnB1boIJksiUq8ATjF6F26B7ytBhE89SvolmA5nMkLiB-yusbSMY0ccxRWpPmX4MJ2yKuM6Sr6L6Dxrw_FWBHU1ThnnxUw') // eslint-disable-line max-len
```
Comment (dimak1) [inline ppr-ui/tests/jest.setup.ts:11]: 👍🏻

**Review by dimak1 — APPROVED** [review]
LGTM!

File: ppr-ui/jest.config.js (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/jest.config.js:9]: Nearly infinite, was doing some testing haha

**Review by RuoxuanPengBC — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1565-s448vnv5.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1565-s448vnv5.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1565-s448vnv5.web.app

**Pull request merged**